lib/plugin.js

```diff
@@ -106,12 +106,17 @@ WebpackMultiOutput.prototype.apply = function (compiler) {
           _this.options.values.forEach(function (value) {
             var basename = _path2.default.basename(file, '.js');
             var filename = value + '.' + basename + '.js';
+            var shouldAddSourceMappingURL = !/manifest/.test(filename);
 
             _this.processSource(value, (0, _lodash2.default)(source), function (result) {
               _this.log('Add asset ' + filename);
               compilation.assets[filename] = result;
+              if (!/manifest/.test(filename)) {
```

Comment:
can we remove this special-case? it should be okay to add sourceMaps to everything. this name ("manifest") is specific to what we've named it.

Reply:
alright. trying this out